src/spring-cloud/azext_spring_cloud/custom.py

```diff
@@ -611,10 +620,11 @@ def app_set_deployment(cmd, client, resource_group, service, name, deployment):
 def deployment_create(cmd, client, resource_group, service, app, name,
                       skip_clone_settings=False,
                       version=None,
-                      jar_path=None,
+                      artifact_path=None,
```

Comment:
This will ignore `jar_path` parameter input in `az spring-cloud app deployment create` command. 
If this is on purpose, _param need to be updated as well, and the version bump up should be a major version, and add PM for awareness.

Reply:
At internal function level, we unified all jar-path to artifact-path, but in parameter, we still keep the old one and add new one.